src/main/java/com/spikes2212/path/Path.java

```diff
@@ -0,0 +1,109 @@
+package com.spikes2212.path;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Path {
+    private List<Waypoint> points;
+    public Path(int middlePoints, double data_weight, double smooth_weight, double tolerance,
```

Comment:
please add a small bit of documentation that explains the proportions of the different weights passed to the path.